pytket/extensions/cutensornet/structured_state/mps.py

```diff
@@ -288,6 +288,74 @@ def apply_qubit_relabelling(self, qubit_map: dict[Qubit, Qubit]) -> MPS:
         self._logger.debug(f"Relabelled qubits... {qubit_map}")
         return self
 
+    def add_qubit(self, new_qubit: Qubit, position: int, state: int = 0) -> MPS:
+        """Adds a qubit on state ``|0>`` at the specified position.

```

Comment:
I am not sure about comment, why is there a state argument when the state is always |0> ?

Reply:
Oops, it used to always be |0>, but then I added the `state` option. Good catch, will fix.

Reply:
Done